### PR TITLE
#167373681 A user should be able to see a home page

### PR DESCRIPTION
**Pull request by JamesMudidi**
## Description ##
- A user needs to be welcomed by a friendly home page.

## Type of change ##
Please select the relevant option
- [ ] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [x] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[#167373681](https://www.pivotaltracker.com/story/show/167373681)

## Screenshots ##
Screenshot 2019-08-01 at 6 41 29 AM

### CODE REVIEW

**Review by CryceTruly — COMMENTED** [review]
Hello @JamesMudidi, could you please fix the failing tests such we can be able to run and understand the test behavior.

File: src/app/pages/home/home.component.html [hunk 1]
```diff
@@ -1,65 +1,201 @@
...
     </div>
   </div>
-  <button [disabled]="userForm.form.valid" class="btn-form btn-form-gold">Sign In</button>
-  <button [disabled]="userForm.form.valid" class="btn-social btn-social-facebook">facebook</button>
-  <button [disabled]="userForm.form.valid" class="btn-social btn-social-google">google</button>
-  <button [disabled]="userForm.form.valid" class="btn-social btn-social-twitter">twitter</button>
-  <button [disabled]="userForm.form.valid" class="btn-form btn-form-blue">Pay Now</button>
-</form>
+</div>
+<app-footer></app-footer>
```
Comment (CryceTruly) [inline src/app/pages/home/home.component.html:360]: @JamesMudidi let us remember to leave a blank line here

File: src/app/pages/home/home.component.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+hr.br-line {
+    background: linear-gradient(to right, var(--color-primary) 25%, var(--color-primary-dark) 75%);
+    padding: 0.50px;
+    margin-left: 35%;
+    margin-right:35%;
+    width: 30%;
+  
+  }
+
```
Comment (CryceTruly) [inline src/app/pages/home/home.component.scss]: @JamesMudidi let us remember to leave a blank line here

**Review by CryceTruly — CHANGES_REQUESTED** [review]

**Review by Victoradukwu — CHANGES_REQUESTED** [review]
@JamesMudidi Please work on the coverage drop and the merge conflicts.

**Review by Genza999 — APPROVED** [review]
LGTM, Great work James, Like the responsiveness on the UI.

File: src/app/components/footer/footer.component.html [hunk 3]
```diff
@@ -46,4 +46,4 @@
     <!-- end /.container -->
   </div>
   <!-- end /.footer-big -->
-</footer>
+</footer>
```
Comment (CryceTruly) [inline src/app/components/footer/footer.component.html]: Good work @JamesMudidi, Please leave a blank line here

File: src/app/pages/home/home.component.scss [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    font-size: 4rem;
+    color: whitesmoke;
+}
+
+.logo {
+    width: 3rem;
+    height: 3rem;
+}
+
+.page-header {
```
Comment (CryceTruly) [inline src/app/pages/home/home.component.scss:98]: Good work @JamesMudidi, I notice the background image takes really long to load, could you minify it and use a minified version of it for faster rendering., You can check if this website can help https://tinypng.com/

**Review by CryceTruly — CHANGES_REQUESTED** [review]

File: src/app/layouts/auth-layout/auth-layout.module.ts [hunk 5]
```diff
@@ -1,22 +1,20 @@
...
-import {LoginComponent} from '../../pages/login/login.component';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { RouterModule } from '@angular/router';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NgxSpinnerModule } from 'ngx-spinner';
+import { AuthLayoutRoutes } from './auth-layout.routing';
+import { LoginComponent } from '../../pages/login/login.component';
 import { LoginFormComponent } from '../../pages/login/login-form/login-form.component';
```
Comment (MandelaK) [inline src/app/layouts/auth-layout/auth-layout.module.ts]: Please use absolute imports ... Relative imports are really hard to read and hard to edit should there be changes required later on

**Review by MandelaK — COMMENTED** [review]
Nice implementation James, I would however suggest that you reduce using inline styles and instead give your elements classes for styling just so that concerns are separated ... you could also make use of bootstrap classes to change things like margins and padding instead of the inline styles ...

Also, please avoid using relative imports because they are really hard to read and edit should changes be necessary

**Review by CryceTruly — APPROVED** [review]
LGTM

**Review by MandelaK — APPROVED** [review]
Good work James ... The UI looks stunning

**Review by Victoradukwu — APPROVED** [review]
LGTM

**Pull request merged**